Godeps/_workspace/src/github.com/opencontainers/specs/spec_linux.go

```diff
@@ -28,6 +28,8 @@ type Linux struct {
 	Capabilities []string `json:"capabilities"`
 	// Devices are a list of device nodes that are created and enabled for the container.
 	Devices []string `json:"devices"`
+	// RootfsPropagation is the rootfs mount propagation mode for the container.
+	RootfsPropagation string `json:"rootfsPropagation"`
```

Comment:
there is a [separate PR](https://github.com/opencontainers/specs/pull/71) against specs